Tasks/ArchiveFiles/task.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "d8b84976-e99a-4b86-b885-4849694435b0",
+  "name": "ArchiveFiles",
+  "friendlyName": "Archive Files",
+  "description": "Archive files using a variety of compression formats such as .7z, .rar, .tar.gz, and .zip.",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "[More Information](TODO)",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [],
+  "version": {
+    "Major": "1",
+    "Minor": "0",
+    "Patch": "0"
+  },
+  "groups": [
+        {
+          "name": "archive",
+          "displayName": "Archive",
+          "isExpanded": true
+        }
+    ],
+  "instanceNameFormat": "Archive files $(message)",
+  "inputs": [
+    {
+      "name": "rootFolder",
+      "type": "filePath",
+      "label": "Root folder (or file) to archive",
+      "defaultValue": "$(Build.BinariesDirectory)",
+      "required": true,
+      "helpMarkDown": "The root folder to add to the archive.  Everything under this folder will be added to the resulting archive."
+    },
+    {
+      "name": "includeRootFolder",
+      "type": "boolean",
+      "label": "Include root folder in archive paths",
+      "defaultValue": true,
+      "required": true,
+      "helpMarkDown": "If set, the root folder will be included in all filepaths within the archive, otherwise all file paths will start one level lower.  <br/><br/>For example, suppose the selected Root Folder is <b>`/home/user/output/classes/`</b>, and contains <b>`com/acme/Main.class`</b>. <ul><li>If set, the resulting archive would contain <b>`classes/com/acme/Main.class`</b>.</li><li>If not set, the resulting archive would contain <b>`com/acme/Main.class`</b>.</li></ul>"
+    },
+    {
+      "name": "archiveType",
+      "type": "pickList",
+      "label": "Archive type",
+      "required": true,
+      "defaultValue": "default",
+      "helpMarkDown": "Specify the compression scheme used.  To create a 'foo.jar', for example, choose the 'zip' for the compression, and specify 'foo.jar' as the archive file to create.  For all tar files (including compressed ones), choose 'tar'.",
```

Comment:
"To create **a** 'foo.jar', for example, choose **the** 'zip'"
In my opinion, this would be easier to read as:
"To create 'foo.jar', for example, choose 'zip'"